src/hub.html

```diff
@@ -1264,7 +1259,6 @@
                 visibility-by-path="path: /actions/leftHand/matrix"
                 hand-controls2="left"
                 teleporter="start: /actions/leftHand/startTeleport; confirm: /actions/leftHand/stopTeleport; collisionEntities: [nav-mesh]"
-                matrix-auto-update
```

Comment:
**CursorPoseTrackingSystem** should be setting the dirty flag for this because it has the `track-pose` component.